File: src/main/java/br/com/jiratorio/domain/vo/DynamicFieldConfig.java
package br.com.jiratorio.domain.vo;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.io.Serializable;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DynamicFieldConfig implements Serializable {
    private static final long serialVersionUID = -8677745120309321988L;

    private String name;

    private String field;

}

File: src/main/kotlin/br/com/jiratorio/domain/dynamicfield/DynamicFieldConfig.kt
package br.com.jiratorio.domain.dynamicfield

import java.io.Serializable

data class DynamicFieldConfig(
    var name: String,
    var field: String
) : Serializable {
    companion object {
        private const val serialVersionUID = -8677745120309321988L
    }

}
